libclc/CMakeLists.txt

```diff
@@ -27,31 +36,51 @@ set( LIBCLC_TARGETS_TO_BUILD "all"
 
 option( ENABLE_RUNTIME_SUBNORMAL "Enable runtime linking of subnormal support." OFF )
 
-find_package(LLVM REQUIRED HINTS "${LLVM_CMAKE_DIR}")
-include(AddLLVM)
+if(CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
```

Comment:
Maybe this should be an `OR` with `LIBCLC_STANDALONE_BUILD`, in order to have projects using libclc with `ExternalProject_Add` using that path? I'm not too sure

Reply:
Yeah, I think you're right. I'm not 100%, but other LLVM sub-projects do it so I think we should too.